### PR TITLE
Italian locale for descendant books

### PR DESCRIPTION
**Pull request by giansalvo**
I'm submitting the Italian translation file for DescendantBooks addon.

### CODE REVIEW

**Comment by prculley** [comment]
I manually merged this into both gramps50 and master branch.  Rebuilt and listed as well.

**Comment by tosky** [comment]
But I didn't even have time to look at it...

**Comment by Nick-Hall** [comment]
@prculley The lead translator should be given the opportunity to review the PR.

Perhaps we should keep a list of GitHub user names on the wiki [Team](https://gramps-project.org/wiki/index.php/Team#Translators) page?  We could then send them a reminder.

File: DescendantBooks/po/it-local.po [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+"vedi resoconto: %(report)s, rif: %(reference)s : %(person)s & %(spouse)s"
+
+#: DescendantBooks/DescendantBookReport.py:319
+#, python-format
+msgid "%s sp."
+msgstr "%s sp."
+
+#: DescendantBooks/DescendantBookReport.py:369
+msgid "Descendants Report"
+msgstr "Resoconto dei Discendenti"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: "Resoconto dei discendenti" (not sure how much this differs from "Descendant Report" without the "s")
In general, please don't use The CamelCase Writing.

File: DescendantBooks/po/it-local.po [hunk 2]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DescendantBookReport.py:374
+#: DescendantBooks/DetailedDescendantBookReport.py:167
+#, python-format
+msgid "Person %s is not in the Database"
+msgstr "La Persona %s non è in archivio"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: "La persona %s ..."  (lowercase)

File: DescendantBooks/po/it-local.po [hunk 3]
```diff
@@ -0,0 +1,739 @@
...
+#: DescendantBooks/DescendantBookReport.py:418
+#: DescendantBooks/DetailedDescendantBookReport.py:409
+#, python-format
+msgid "Writing %s reports..."
+msgstr "Scrittura di %s resoconti..."
+
+#: DescendantBooks/DescendantBookReport.py:437
+#, python-format
+msgid "%s. Descendants of %s"
+msgstr "%s Discendenti di %s"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: lowercase: 
"%s discendenti di %s"

File: DescendantBooks/po/it-local.po [hunk 4]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DescendantBookReport.py:439
+#, python-format
+msgid "Descendants of %s"
+msgstr "Discendenti di %s"
+
+#: DescendantBooks/DescendantBookReport.py:459
+msgid "Descendant Report"
+msgstr "Resoconto dei Discendenti"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: This is "Resoconto discendenza" in Gramps.

File: DescendantBooks/po/it-local.po [hunk 5]
```diff
@@ -0,0 +1,739 @@
...
+#: DescendantBooks/DescendantBookReport.py:476
+#: DescendantBooks/DetailedDescendantBookReport.py:602
+#, python-format
+msgid "%d. %s"
+msgstr "%d. %s"
+
+#: DescendantBooks/DescendantBookReport.py:501
+#: DescendantBooks/DetailedDescendantBookReport.py:1325
+msgid "Report Options"
+msgstr "Opzioni del Resoconto"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: Opzioni del resoconto

File: DescendantBooks/po/it-local.po [hunk 6]
```diff
@@ -0,0 +1,739 @@
...
+msgid "Numbering system"
+msgstr "Sistema di numerazione"
+
+#: DescendantBooks/DescendantBookReport.py:528
+msgid "Simple numbering"
+msgstr "Numerazione semplice"
+
+#: DescendantBooks/DescendantBookReport.py:529
+msgid "de Villiers/Pama numbering"
+msgstr "numerazione de Villiers/Pama"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: Numerazione de Villiers/Pama

File: DescendantBooks/po/it-local.po [hunk 7]
```diff
@@ -0,0 +1,739 @@
...
+msgstr "Lo stile usato per visualizzare il livello %d."
+
+#: DescendantBooks/DescendantBookReport.py:611
+#, python-format
+msgid "The style used for the spouse level %d display."
+msgstr "Lo stile usato per visualizzare il livello %d del coniuge."
+
+#: DescendantBooks/DescendantBooks.gpr.py:20
+msgid "Descendant Book"
+msgstr "Resoconto dei discendenti"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: For consistency, this should be "Libro discendenza"

File: DescendantBooks/po/it-local.po [hunk 8]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DescendantBooks.gpr.py:21
+msgid "Produces one or more descendant reports based on a supplied query."
+msgstr ""
+"Fornisce uno o più resoconti dei discendenti basati sulla richiesta "
+"indicata."
+
+#: DescendantBooks/DescendantBooks.gpr.py:37
+msgid "Detailed Descendant Book"
+msgstr "Resoconto dei discendenti dettagliato"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: See above

File: DescendantBooks/po/it-local.po [hunk 9]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:347
+#, python-format
+msgid "Generating %s report references..."
+msgstr "Generazione di %s riferimenti del resoconto..."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:441
+#, python-format
+msgid "%(report_count)s. Descendant Report for %(person_name)s"
+msgstr "%(report_count)s. Resoconto dei discendenti di %(person_name)s"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: %(report_count)s resoconto discendenza di %(person_name)s

File: DescendantBooks/po/it-local.po [hunk 10]
```diff
@@ -0,0 +1,739 @@
...
+msgstr "Generazione %d"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:585
+msgid "Detailed Descendant Report"
+msgstr "Resoconto dettagliato dei discendenti"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:682
+#, python-format
+msgid "See Report : %s, Generation : %s, Person : %s"
+msgstr "Vedi resoconto: %s, Generazione: %s, Persona: %s"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: Vedi resoconto: %s, generazione: %s, persona: %s

File: DescendantBooks/po/it-local.po [hunk 11]
```diff
@@ -0,0 +1,739 @@
...
+msgid "Descendant Report for %(person_name)s"
+msgstr "Resoconto dei discendenti per %(person_name)s"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:458
+#, python-format
+msgid "Generation %d"
+msgstr "Generazione %d"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:585
+msgid "Detailed Descendant Report"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: See above

File: DescendantBooks/po/it-local.po [hunk 12]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:441
+#, python-format
+msgid "%(report_count)s. Descendant Report for %(person_name)s"
+msgstr "%(report_count)s. Resoconto dei discendenti di %(person_name)s"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:445
+#, python-format
+msgid "Descendant Report for %(person_name)s"
+msgstr "Resoconto dei discendenti per %(person_name)s"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 13]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:963
+#, python-format
+msgid "Spouse: %s"
+msgstr "Coniuge: %s"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:965
+#, python-format
+msgid "Relationship with: %s"
+msgstr "Relazione cont: %s"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: "Relazione con: %s"

File: DescendantBooks/po/it-local.po [hunk 14]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1015
+#, python-format
+msgid "Children of %(mother_name)s and %(father_name)s"
+msgstr "Figlio di %(mother_name)s e %(father_name)s"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1073
+#, python-format
+msgid "Notes for %(mother_name)s and %(father_name)s:"
+msgstr "Annotazioni di %(mother_name)s e %(father_name)s:"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: "Note per %(mother_name)s and %(father_name)s:"

File: DescendantBooks/po/it-local.po [hunk 15]
```diff
@@ -0,0 +1,739 @@
...
+#: DescendantBooks/DetailedDescendantBookReport.py:1096
+#: DescendantBooks/DetailedDescendantBookReport.py:1119
+#, python-format
+msgid "More about %(mother_name)s and %(father_name)s:"
+msgstr "Altro su %(mother_name)s e %(father_name)s:"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1201
+#, python-format
+msgid "Notes for %s"
+msgstr "Annotazioni per %s"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: Note per

File: DescendantBooks/po/it-local.po [hunk 16]
```diff
@@ -0,0 +1,739 @@
...
+msgid "Whether to start a new page before the end notes."
+msgstr "Indica se iniziare una nuova pagina prima delle annotazioni."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1378
+msgid "Content"
+msgstr "Contenuto"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1380
+msgid "Use callname for common name"
+msgstr "Usa il soprannome come nome comune"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: I used pseudonimo for callname; soprannome is used for nickname.
We can argue whether the two terms should be switched, but for now they should be consistent.

File: DescendantBooks/po/it-local.po [hunk 17]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1381
+msgid "Whether to use the call name as the first name."
+msgstr "Indica se usare il soprannome come primo nome."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 18]
```diff
@@ -0,0 +1,739 @@
...
+msgid "Compute death age"
+msgstr "Calcola l'età alla morte"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1394
+msgid "Whether to compute a person's age at death."
+msgstr "Indica se calcolare l'età di una persona alla morte oppure no."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1397
+msgid "Omit duplicate ancestors"
+msgstr "Omette gli antenati duplicati"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: I would use the same string as in Gramps ("Omette gli ascendenti duplicati"). You can probably use the translation memory feature of your translation program.

File: DescendantBooks/po/it-local.po [hunk 19]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1398
+msgid "Whether to omit duplicate ancestors."
+msgstr "Indica se omettere gli antenati duplicati oppure no."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above: ancestor -> ascendente

File: DescendantBooks/po/it-local.po [hunk 20]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1401
+msgid "Use complete sentences"
+msgstr "Usa frasi complete"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1403
+msgid "Whether to use complete sentences or succinct language."
+msgstr "Indica se usa frasi complete oppure linguaggio succinto."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: The same sentence is "Indica se usare frasi complete oppure un linguaggio sintetico."

File: DescendantBooks/po/it-local.po [hunk 21]
```diff
@@ -0,0 +1,739 @@
...
+"figli."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1412
+#: DescendantBooks/DetailedDescendantBookReport.py:1413
+msgid "Include"
+msgstr "Includi"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1415
+msgid "Include notes"
+msgstr "Includi le annotazioni"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: Please use "note". "annotazioni" is used for "remarks"

File: DescendantBooks/po/it-local.po [hunk 22]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1416
+msgid "Whether to include notes."
+msgstr "Indica se includere o meno le annotazioni."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 23]
```diff
@@ -0,0 +1,739 @@
...
+msgid "Whether to include images."
+msgstr "Indica se includere o meno le immagini."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1427
+msgid "Include alternative names"
+msgstr "Includi i nomi alternativi"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1428
+msgid "Whether to include other names."
+msgstr "Indica se includere oppure no i nomi alternativi."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: "Indica se includere gli altri nomi."

File: DescendantBooks/po/it-local.po [hunk 24]
```diff
@@ -0,0 +1,739 @@
...
+msgid "Include sources"
+msgstr "Includi le fonti"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1440
+msgid "Whether to include source references."
+msgstr "Indica se includere o meno i riferimenti alle fonti."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1443
+msgid "Include sources notes"
+msgstr "Includi le annotazioni delle fonti"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: note

File: DescendantBooks/po/it-local.po [hunk 25]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1444
+msgid ""
+"Whether to include source notes in the Endnotes section. Only works if "
+"Include sources is selected."
+msgstr ""
+"Indica se includere le annotazioni delle fonti nella sezione delle "
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: note
endnotes is "note a piè di pagina" (and it should not be Camel Case)

File: DescendantBooks/po/it-local.po [hunk 26]
```diff
@@ -0,0 +1,739 @@
...
+msgid "Include spouse reference"
+msgstr "Includi il riferimento al coniuge"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1453
+msgid "Whether to include reference to spouse."
+msgstr "Indica se includere o meno il riferimento al coniuge."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1456
+msgid "Include sign of succession ('+') in child-list"
+msgstr "Include il segno di successione ('+') nella lista dei figli"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: missing consistency

File: DescendantBooks/po/it-local.po [hunk 27]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1464
+msgid ""
+"Whether to include the path of descendancy from the start-person to each "
+"descendant."
+msgstr "Indica se includere o meno il percorso fino alla persona iniziale."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1468
+msgid "Include Index of Names"
+msgstr "Includi l'Indice dei Nomi"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: lowecase

File: DescendantBooks/po/it-local.po [hunk 28]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1469
+msgid "Whether to include the index of Names at the end of the report."
+msgstr "Indica se includere o meno l'Indice dei Nomi"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: lowecase

File: DescendantBooks/po/it-local.po [hunk 29]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1473
+msgid "Include index of Dates"
+msgstr "Includi l'indice delle Date"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: lowecase

File: DescendantBooks/po/it-local.po [hunk 30]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1474
+msgid "Whether to include the index of Dates at the end of the report."
+msgstr ""
+"Indica se includere o meno l'Indice delle Date alla fine del resoconto."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: lowercase

File: DescendantBooks/po/it-local.po [hunk 31]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1478
+msgid "Include index of Places"
+msgstr "Includi l'indice dei Luoghi"
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 32]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1479
+msgid "Whether to include the index of Places at the end of the report."
+msgstr ""
+"Indica se includere o meno l'Indice dei Luoghi alla fine del resoconto."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 33]
```diff
@@ -0,0 +1,739 @@
...
+msgid "Missing information"
+msgstr "Informazione mancante"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1488
+msgid "Replace missing places with ______"
+msgstr "Sostituisci i luoghi mancanti con _______"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1489
+msgid "Whether to replace missing Places with blanks."
+msgstr "Indica se sostituire o meno i Luoghi mancanti con spazi vuoti."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 34]
```diff
@@ -0,0 +1,739 @@
...
+msgid "The style used for the children list title."
+msgstr "Lo stile usato per il titolo della lista dei figli."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1546
+msgid "The style used for the children list."
+msgstr "Lo stile usato per la lista dei figli."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1556
+msgid "The basic style used for Note header."
+msgstr "Lo stile di base usato per l'intestazione delle Annotazioni."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: note

File: DescendantBooks/po/it-local.po [hunk 35]
```diff
@@ -0,0 +1,739 @@
...
+msgstr "Lo stile di base usato per il testo."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1573
+msgid "The style used for the first personal entry."
+msgstr "Lo stile usato la prima registrazione personale."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1583
+msgid "The style used for the More About header and for headers of mates."
+msgstr ""
+"Lo stile usato per l'intestazione di 'Altro a proposito di' e per "
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: More About is "Altre informazioni su"

File: DescendantBooks/po/it-local.po [hunk 36]
```diff
@@ -0,0 +1,739 @@
...
+msgstr "Lo stile usato per i dettagli addizionali."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1604
+#: DescendantBooks/DetailedDescendantBookReport.py:1614
+msgid "The style used for note detail data."
+msgstr "Lo stile usato per i dettagli delle annotazioni."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1624
+msgid "The style used for the date in the Index of Dates."
+msgstr "Lo stile usato per le date nell'Indice delle Date."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 37]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1634
+msgid "The style used for the events in the Index of Dates."
+msgstr "Lo stile usato per gli eventi nell'Indice delle Date."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 38]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1644
+msgid "The style used for the header in the Index of Names."
+msgstr "Lo stile usato per le intestazioni nell'Indice dei Nomi."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 39]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1654
+msgid "The style used for the references in the Index of Names."
+msgstr "Lo stile usato per i riferimenti nell'Indice dei Nomi."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 40]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1664
+msgid "The style used for the header in the Index of Places."
+msgstr "Lo stile usato per le intestazioni nell'Indice dei Luoghi."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po [hunk 41]
```diff
@@ -0,0 +1,739 @@
...
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1675
+msgid "The style used for the references in the Index of Places."
+msgstr "Lo stile usato per i riferimenti nell'Indice dei Luoghi."
```
Comment (tosky) [inline DescendantBooks/po/it-local.po]: see above

File: DescendantBooks/po/it-local.po (hunk 1 above)
Comment (giansalvo) [reply DescendantBooks/po/it-local.po]: ok

**Comment by giansalvo** [comment]
@tosky 
I amended the po files as per your review. I'm new with GitHub workflow, how should I go on to send the new file? Should I open a new PR or is there a way to use this one?
thanks

**Comment by prculley** [comment]
You add a new commit with the changes to your branch.  I would then merge the three commits together into a single commit, since we don't really need all the details of your progress forever.  I use
git rebase -i master
which starts an interactive rebase on your text editor, you can use the instructions to rename the first commit to something appropriate, and 'f' to merge in the next commits.  The 'master' can be any tag that points to the commit before the ones of interest, you can also use a commit hash from git log or as seen when you look at the 'gitk' graphical display of your git repos.
Then 
git push -f origin italian_locale_for_descendant_books
The -f is force push, needed when your local branch changes conflict with the remote branch, would would be true due to the rebase.  The 'origin' is whatever you have named the remote repo (usually origin or upstream).

The push will update your Github copy, which in turn will automatically update the PR.

**Comment by giansalvo** [comment]
> Since this PR is closed, you should probably start a new PR. It may be possible to reopen the PR, but I suspect that it may have issues since the prior commits are already merged.

@prculley 
Ok, thanks. Should I open the new PR against master branch or maintenance/gramps50 (see #188) or something else?

**Comment by prculley** [comment]
For the time being, if the translations are out of date on the gramps50 branch, make your changes there.  They will be merged into master the next time we do a periodic general merge.  At some point when master has begun to diverge to a greater extent than present, we will call for translators to look at master.  If you make changes to master only, it will take quite some time before anyone can see them, but the gramps50 branch will probably see another release in a month or so.

**Pull request closed without merging**